typescript/example_code/cdk/my_ecs_construct-stack.ts

```diff
@@ -54,7 +54,7 @@ export class MyEcsConstructStack extends core.Stack {
       publicLoadBalancer: true // Default is false
     });
     // snippet-end:[cdk.typescript.my_ecs_construct.create_fargate_service]
-    // snippet-end:[cdk.typescript.my_ecs_construct-stack.class]
   }
 }
 // snippet-end:[cdk.typescript.my_ecs_construct-stack]
+// snippet-end:[cdk.typescript.my_ecs_construct-stack.class]

```

Comment:
These nest and since `cdk.typescript.my_ecs_construct-stack` starts on line 25 while `cdk.typescript.my_ecs_construct-stack.class` starts on line 33, its probably best to have `cdk.typescript.my_ecs_construct-stack.class` come first.

Reply:
Learned something new! Thanks; fixed.